pkgs/shells/zsh/zsh-fzf-tab/default.nix

```diff
@@ -33,6 +33,7 @@ in stdenv.mkDerivation rec {
      cp -r lib ${INSTALL_PATH}/lib
      install -D fzf-tab.zsh ${INSTALL_PATH}/fzf-tab.zsh
      install -D fzf-tab.plugin.zsh ${INSTALL_PATH}/fzf-tab.plugin.zsh
+  '' + lib.optionalString (!stdenv.isDarwin) ''
```

Comment:
shouldn't there be a corresponding .dylib built ?

Reply:
I agree. If macos doesn't have the so file, which dynamic lib does it use? Whether, your fzf-tab work with the built lib?

Reply:
I tried to build on x64 darwin seems to be missing steps
https://logs.ofborg.org/?key=nixos/nixpkgs.270631&attempt_id=f2cb2333-5a0b-42cc-8f8f-010a9eb73285

there is no linux build because they are cached, but comparing darwin x64 to local linux build there seem to be steps missing (also, in the .nix file for postConfigure, probably should add `--prefix=$out` to get rid of usr/local references

darwin x64
```console
Updated `zsh.mdh'.
make -f Makemod prefix='/usr/local' exec_prefix='/usr/local' bindir='/usr/local/bin' libdir='/usr/local/lib' MODDIR='/usr/local/lib/zsh/5.3.1-dev-0' infodir='/usr/local/share/info' mandir='/usr/local/share/man' datadir='/usr/local/share' fndir='/usr/local/share/zsh/5.3.1-dev-0/functions' htmldir='/usr/local/share/zsh/htmldoc' runhelpdir='/usr/local/share/zsh/5.3.1-dev-0/help' CC='clang' CPPFLAGS='' DEFS='-DHAVE_CONFIG_H' CFLAGS='-Wall -Wmissing-prototypes -O2' LDFLAGS='-Wl,-x' EXTRA_LDFLAGS='-rdynamic' DLCFLAGS='-fno-common' DLLDFLAGS='-bundle -flat_namespace -undefined suppress' LIBLDFLAGS='' EXELDFLAGS='' LIBS='-ldl -lncursesw -lm  -lc' DL_EXT='so' DLLD='clang' AWK='gawk' ANSI2KNR=': ansi2knr' YODL=': yodl ' YODL2TXT=': yodl2txt' YODL2HTML=': yodl2html' FUNCTIONS_INSTALL='' tzsh='zsh' prep
make[3]: Entering directory '/private/tmp/nix-build-zsh-fzf-tab-unstable-2023-06-11.drv-0/source/modules/Src'
make[3]: Leaving directory '/private/tmp/nix-build-zsh-fzf-tab-unstable-2023-06-11.drv-0/source/modules/Src'
echo 'timestamp for *.mdd files' > ../Src/modules.stamp
make[2]: Leaving directory '/private/tmp/nix-build-zsh-fzf-tab-unstable-2023-06-11.drv-0/source/modules/Src'
make[2]: Entering directory '/private/tmp/nix-build-zsh-fzf-tab-unstable-2023-06-11.drv-0/source/modules/Src'
make[2]: Leaving directory '/private/tmp/nix-build-zsh-fzf-tab-unstable-2023-06-11.drv-0/source/modules/Src'
make[1]: Leaving directory '/private/tmp/nix-build-zsh-fzf-tab-unstable-2023-06-11.drv-0/source/modules/Src'
/private/tmp/nix-build-zsh-fzf-tab-unstable-2023-06-11.drv-0/source

```

linux x64 (from nix-build -A zsh-fzf-tab --check).
```console
Updated `zsh.mdh'.
make -f Makemod prefix='/usr/local' exec_prefix='/usr/local' bindir='/usr/local/bin' libdir='/usr/local/lib' MODDIR='/usr/local/lib/zsh/5.3.1-dev-0' infodir='/usr/local/share/info' mandir='/usr/local/share/man' datadir='/usr/local/share' fndir='/usr/local/share/zsh/5.3.1-dev-0/functions' htmldir='/usr/local/share/zsh/htmldoc' runhelpdir='/usr/local/share/zsh/5.3.1-dev-0/help' CC='gcc' CPPFLAGS='' DEFS='-DHAVE_CONFIG_H' CFLAGS='-Wall -Wmissing-prototypes -O2' LDFLAGS='' EXTRA_LDFLAGS='-rdynamic' DLCFLAGS='-fPIC' DLLDFLAGS='-shared' LIBLDFLAGS='-s' EXELDFLAGS='-s' LIBS='-ldl -lncursesw -lrt -lm  -lc' DL_EXT='so' DLLD='gcc' AWK='gawk' ANSI2KNR=': ansi2knr' YODL=': yodl ' YODL2TXT=': yodl2txt' YODL2HTML=': yodl2html' FUNCTIONS_INSTALL='' tzsh='zsh' prep
make[3]: Entering directory '/build/source/modules/Src'
creating Src/aloxaf/Makefile.in
config.status: creating Src/aloxaf/Makefile
make[4]: Entering directory '/build/source/modules/Src/aloxaf'
make[4]: Leaving directory '/build/source/modules/Src/aloxaf'
make[3]: Leaving directory '/build/source/modules/Src'
echo 'timestamp for *.mdd files' > ../Src/modules.stamp
make[3]: Entering directory '/build/source/modules/Src/aloxaf'
make[4]: Entering directory '/build/source/modules/Src/aloxaf'
make[4]: Leaving directory '/build/source/modules/Src/aloxaf'
Updated `fzftab.mdh'.
echo 'timestamp for fzftab.mdh against fzftab.mdd' > fzftab.mdhs
gawk -f ../../Src/makepro.awk fzftab.c Src/aloxaf > fzftab.syms
gawk: ../../Src/makepro.awk:124: warning: regexp escape sequence `\=' is not a known regexp operator
(echo '/* Generated automatically */'; sed -n '/^E/{s/^E//;p;}' < fzftab.syms) \
        > fzftab.epro
(echo '/* Generated automatically */'; sed -n '/^L/{s/^L//;p;}' < fzftab.syms) \
        > `echo fzftab.epro | sed 's/\.epro$/.pro/'`
make[4]: Entering directory '/build/source/modules/Src/aloxaf'
make[4]: Leaving directory '/build/source/modules/Src/aloxaf'
Updated `fzftab.mdh'.
make[3]: Leaving directory '/build/source/modules/Src/aloxaf'
make[2]: Leaving directory '/build/source/modules/Src'
make[2]: Entering directory '/build/source/modules/Src'
make[3]: Entering directory '/build/source/modules/Src/aloxaf'
gcc -c -I. -I../../Src -I../../Src -I../../Src/Zle -I.  -DHAVE_CONFIG_H -DMODULE -Wall -Wmissing-prototypes -O2 -fPIC -o fzftab..o fzftab.c
fzftab.c: In function 'fzf_tab_colorize':
fzftab.c:353:13: warning: ignoring return value of 'realpath' declared with attribute 'warn_unused_result' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result-Wunused-result8;;]
  353 |             realpath(file, tmp);
      |             ^~~~~~~~~~~~~~~~~~~
rm -f fzftab.so
gcc  -s -shared -o fzftab.so   fzftab..o    -ldl -lncursesw -lrt -lm  -lc 
make[3]: Leaving directory '/build/source/modules/Src/aloxaf'
make[2]: Leaving directory '/build/source/modules/Src'
make[1]: Leaving directory '/build/source/modules/Src'
/build/source

```